cli/spec/kontena/cli/nodes/ssh_command_spec.rb

```diff
@@ -16,9 +16,18 @@
     allow(client).to receive(:get).with('nodes/test-grid/test-node').and_return(node)
   end
 
-  it "fails if using both --any and a node name as a command" do
-    expect(subject).to_not receive(:exec)
-    expect{subject.run(['--any', 'ls', '-l'])}.to exit_with_error.and output(/Cannot combine --any with a node name/).to_stderr
+  context '--any' do
```

Comment:
If you want to group up the examples that are using `--any`, then this should probably be `describe`, not `context`. They're technically the same thing, but `context` should be for what the state the server nodes are in, and `describe` for what functionality the specs test: http://lmws.net/describe-vs-context-in-rspec

```ruby
describe '--any' do
  it 'fails if given a node name' do
    ...
  end
  context 'with all nodes offline' do
    before do
      allow(client).to receive(:get).with(...).and_return(...)
    end

    it 'fails' do
      ...
    end
  end
end
```

The resulting rspec descriptions should read nicely:

* `Kontena::Cli::Nodes::SshCommand --any fails if given a node name`
* `Kontena::Cli::Nodes::SshCommand --any with all nodes offline fails`